cluster/src/dunit/scala/io/snappydata/metrics/SnappyMetricsSystemDUnitTest.scala

```diff
@@ -79,10 +71,10 @@ class SnappyMetricsSystemDUnitTest(s: String)
 
   override def afterClass(): Unit = {
     logInfo((snappyProductDir + "/sbin/snappy-stop-all.sh").!!)
-    s"rm -rf $snappyProductDir/work".!!
-    Files.deleteIfExists(Paths.get(snappyProductDir, "conf", "locators"))
-    Files.deleteIfExists(Paths.get(snappyProductDir, "conf", "leads"))
-    Files.deleteIfExists(Paths.get(snappyProductDir, "conf", "servers"))
+    // s"rm -rf $snappyProductDir/work".!!
```

Comment:
What's the reason to retain the work directory?

Reply:
Mistakenly added comment for these lines. Removed it